starter-code/assets/css/reset.css

```diff
@@ -0,0 +1,137 @@
+/* http://meyerweb.com/eric/tools/css/reset/ 
+   v2.0 | 20110126

```

Comment:
Since we are using code from a template, we do not want a reset. We want to retain the original styling, just modify it to personalize it. Delete this file from your index.html.